corehq/apps/export/models/new.py

```diff
@@ -251,9 +249,6 @@ class ExportInstance(Document):
     # Whether to split multiselects into multiple columns
     split_multiselects = BooleanProperty(default=False)
 
-    # Whether to automatically convert dates to excel dates
-    transform_dates = BooleanProperty(default=False)
```

Comment:
Now that I think of it we have no way to maintain the state of the checkbox
that says to transform all dates so we'd need the property anyways. Agreed,
I think because of the issue I just mentioned I should revert this commit
and if we find ourselves needing more global transforms we can talk about a
strategy

On Tuesday, February 9, 2016, Simon Kelly <notifications@github.com> wrote:

> In corehq/apps/export/models/new.py
> <https://github.com/dimagi/commcare-hq/pull/10243#discussion_r52347084>:
>
> > @@ -251,9 +249,6 @@ class ExportInstance(Document):
> >      # Whether to split multiselects into multiple columns
> >      split_multiselects = BooleanProperty(default=False)
> >
> > -    # Whether to automatically convert dates to excel dates
> > -    transform_dates = BooleanProperty(default=False)
>
> Yea it's an interesting one. In my mind the best option from a data model
> perspective would be to have a 'global_transforms' property at the
> ExportInstance level. Pros being that you don't have to update every column
> to change the value and you don't get duplication in the data. The con is
> that you have to then pass then around in the code. Considering there is
> only 1 global transform option I don't think there is can clear winner. If
> we had more global transform options I would go for the approach described
> above. Fine to leave this as is. One question though, how do you determine
> of the check box needs to be selected when you load the screen for an
> existing export? (Relying via email on my phone so don't have full diff
> handy)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10243/files#r52347084>.
>